src/server/api/utility.ts

```
@@ -0,0 +1,1311 @@
+  /** Operation had no effect */
+  Inconsequential = "Inconsequential",
+}
+
+// -- Response Type --
+
+/** Generic API response type */
+export type apiResponse<T> = {
+  status: apiResponseStatus;
+  type?: apiResponseTypes;
+  message?: string;
+  data?: T extends void | undefined | null ? undefined : T;
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)